View/register_page.php

```diff
@@ -0,0 +1,69 @@
+<!-- HTML part of register page -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Edu+NSW+ACT+Foundation:wght@400;500&family=Poppins:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="Content/css/style_index.css">
+  <title>Register</title>
+</head>
+<body>
+  <div class="container">
+    <h1>Register</h1>
+    <form action="/register" method="POST" enctype="multipart/form-data">
+      <label for="user">Username</label>
+      <span>*</span>
+      <input type="text" id="user" name="user" required>
+      <span class="error">
+        <?php
+          if(isset($GLOBALS["userErr"])) {
+            echo $GLOBALS["userErr"];
+            unset($GLOBALS["userErr"]);
```

Comment:
I have used unset because after submission of the form the router forwards the request to the controller first where it checks if their is POST data(means if the form is submitting any data) then it will take the request to the model part where backend validation is going on. If no POST data is there then it forwards the request to the view login page part where the user has to input data to the field. In this case the invalid data is passed on to the controller where it will be passed again to the model part without correction so after knowing the data is invalid I am unsetting it.

Reply:
When I used session variables earlier for error messages then it created problems like before form is submitted the error message starts to show as those are accessible anywhere so I had to check using isset and after echo I had to unset.
I haven't used without unset for GLOBALS, I will try it.